usr/bin/nxdk-configure

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/sh
+
+#FIXME: add explicit support for SDL / SDL2?
+#       https://wiki.libsdl.org/FAQLinux#how_do_i_add_sdl_to_my_project
+
+export CC=nxdk-cc
+export PKG_CONFIG=nxdk-pkg-config
+
+# Wrapper around autoconf ./configure scripts
+./configure --host=i686-windows
```

Comment:
This should also configure or modify a prefix, so it never installs into the users system. It should install into some nxdk dummy folder probably.

To be honest: I was too scared to try this. `make install` is always evil.